scripts/conda-recipe/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = environ.get('GIT_DESCRIBE_TAG', "0.dirty").replace('-', '.') %}
+
+package:
+  name: modin
+  version: "{{ version }}"
+
+source:
+  path: ../../
+
+build:
+  number: {{ environ.get('GIT_DESCRIBE_NUMBER', 0) }}
+  script: '{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv '
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - ray ==0.8.6
+    - omniscidbe4py
+    - python
+    - pandas ==1.0.5
+    - dask >=2.1.0
+    - distributed >=2.3.2
+
+test:
+  import:
+    - modin
+    - modin.experemental.pandas
```

Comment:
Sorry for not catching this before, should not`modin.experemental.pandas` be `modin.experimental.pandas`?

Reply:
I guess it should, cc @anton-malakhov